app/Providers/RouteServiceProvider.php

```diff
@@ -42,10 +42,7 @@ public function boot()
                 ->middleware('api')
                 ->namespace($this->namespace)
                 ->group(base_path('routes/api.php'));
-//            Route::prefix('customer')
-//                ->middleware('web')
-//                ->namespace($this->namespace)
```

Comment:
please make 'master' branch has default of laravel's base source code, then your custom branch has all your code